charts/rstudio-workbench/values.yaml

```diff
@@ -63,6 +63,30 @@ strategy:
     maxSurge: "100%"
     maxUnavailable: 0
 
+# -- Settings for the Chronicle Agent sidecar container
+chronicleAgent:
+  # -- Whether to enable the Chronicle Agent sidecar container
+  enabled: false
+  # -- An array of maps that is injected as-is into the "env:" component of the container spec
+  env: []
+  # -- An array of maps that is injected as-is into the "volumeMounts" component of the container spec
+  volumeMounts: []
+  # -- If true, the chart will attempt to lookup the Chronicle Server address and version in the cluster
+  autoDiscovery: true
+  image:
+    # -- The registry to use for the Chronicle Agent image
+    registry: ghcr.io
+    # -- The repository to use for the Chronicle Agent image
+    repository: rstudio/chronicle-agent
+    # -- A tag to use for the Chronicle Agent image. If not set, the chart will attempt to look up the version of the deployed Chronicle server in the current namespace.
+    tag: ""
+    # -- The pull policy for the Chronicle Agent image
+    imagePullPolicy: IfNotPresent
+  # -- The address for the Chronicle server. If not set, the chart will attempt to look up the address of the Chronicle Server in the release namespace or the serverNamespace if provided.
+  serverAddress: ""
+  # -- The namespace for the Chronicle server. If not set, the chart will attempt to look up the address of the Chronicle Server in the release namespace.
+  serverNamespace: ""
+
```

Comment:
There should be a way to provide a Workbench API key here as well. This API key is optional for agents running alongside Workbench, but we need to provide a mechanism for a user who wants to to provide one.

Reply:
I added `chronicleAgent.workbenchApiKey` to provide an override.